test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -113,6 +113,89 @@ public static TheoryData MultiAttributeCheckBoxData
             Assert.Null(output.TagName); // Cleared
         }
 
+        [Theory]
+        [InlineData("bad")]
+        [InlineData("notbool")]
+        public void CheckBoxHandlesNonParsableStringsAsBoolsCorrectly(
+            string possibleBool)
+        {
+            // Arrange
+            var originalContent = "original content";
```

Comment:
These values really are constants, and they really aren't variables, so I can't think of even one good reason to avoid using a `const`.